known-issues.yml

```diff
@@ -166,7 +166,13 @@ zib-BodyTemperature:
     Observation.code.coding:BodyTempCode:
       - message: "Constraint failed: sd-pg-13: 'Slice names should be camelCased starting with a lowercase letter.'"
         reason: The slice name "BodyTempCode" is chosen to align with the Vital Signs profile for BodyTemperature.
-        
+
+zib-BowelFunction:
+  zib deviations:
+    Observation.extension:incontinenceMaterial.value[x]:
+      - cardinality: 0..* instead of 0..1
```

Comment:
I think this implementation makes sense, but why haven't we done the same with e.g. the flag-detail extension in zib-Alert, the workflow-reasonReference extension in zib-NutritionAdvice or the condition-dueTo extension in zib-SkinDisorder? In the past we've implemented the zib maximum cardinality constraints to core extensions, even if they represent a more generic concept. Should we fix that, or at least follow the same approach?

Reply:
Er ... I think our insights on this topic has evolved a bit over time. Ideally you'd have a slice and apply the restriction to the slice, but I don't think that's feasible in this case.

So yes, perhaps we should change the other cases as well.

Reply:
Created <https://nictiz.atlassian.net/browse/ZIBFHIR-225>